docs/source/asr/speaker_recognition/models.rst

```diff
@@ -11,6 +11,20 @@ The Checkpoints page also contains benchmark results for the available speaker r
 
 .. _SpeakerNet_model:
 
+TitaNet
+-----------
+
+The model is based on the ContextNet architecture :cite:`sr-models-koluguri2021titanet`  for extracting speaker representations.
```

Comment:
The cite here is supposed to be contextnet?

Reply:
We explain contexnet in titanet paper too.

Reply:
Okay its minor. I thought it would be The model (cite titanet)  is based on the ContextNet architecture (cite contextnet)

Reply:
You should cite ContextNet itself of you are referring to it, doesjt matter if titanet explains